src/clue.js

```diff
@@ -1,28 +1,138 @@
+ },
+  profPlum = {
+   "firstName": "Victor",
+   "lastName": "Plum",
+   "occupation": "Designer",
+   "age": 22,
+   "description": "Billionaire video game designer",
+   image: "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+   "color": "purple"
+ },
+  missScarlet = {
+   "firstName": "Kasandra",
+   "lastName": "Scarlet",
+   "occupation": "Actor",
+   "age": 31,
+   "description": "She is an A-list movie star with a dark past",
+   image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+   "color": "red"
+ },
+  mrsPeacock = {
+   "firstName": "Eleanor",
+   "lastName": "Peacock",
+   "occupation": "Socialité",
+   "age": 31,
+   "description": "She is from a wealthy family and uses her status and money to earn popularity",
+   image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+   "color": "blue"
+ },
+  mrMustard = {
+   "firstName": "Jack",
+   "lastName": "Mustard",
+   "occupation": "Retired Football player",
+   "age": 62,
+   "description": "He is a former football player who tries to get by on his former glory",
+   image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+   "color": "yellow"
+ }
+ ];
+ 
 // Rooms Array
 
-const roomsArray = [];
+const roomsArray = [
+      {name: 'Dinning Room' },
+      {name: 'Conservatory' },
+      {name: 'Kitchen'},
+      {name: 'Study'},
+      {name: 'Library'},
+      {name: 'Billiard Room'},
+      {name: 'Lounge'},
+      {name: 'Ballroom'},
+      {name: 'Hall'},
+      {name: 'Spa'},
+      {name: 'LivingRoom'},
+      {name: 'Observatory'},
+      {name: 'Theater'},
+      {name: 'Guest House'},
+      {name: 'Patio'}, 
+]; 
 
 // Weapons Array
 
-const weaponsArray = [];
+const weaponsArray = [
 
+      {name: 'rope', weight: 10},
+      {name: 'knife', weight: 8},
+      {name: 'candlestic', weight: 2},
+      {name: 'dumbbell', weight: 30},
+      {name: 'poison', weight: 2},
+      {name: 'axe', weight: 15},
+      {name: 'bat', weight: 13},
+      {name: 'trophy', weight: 25},
+      {name: 'pistol', weight: 20}
+];
 
 // ITERATION 2
 
-function selectRandom() {}
+const cartas = [selectRandom, weaponsArray, roomsArray] 
+
+function selectRandom(cartas) {
+ let carta = cartas[Math.floor(Math.random() *cartas.length)];
+ return carta;
+}
+
 
-function pickMystery() {}
+function pickMystery() {
+    let suspect = selectRandom(suspectsArray);
+    let weapon = selectRandom(weaponsArray);
+    let room = selectRandom(roomsArray);
+
+     let combinacion = { suspect, weapon, room }
+
+    return combinacion;
+}
 
 
 // ITERATION 3
 
-function revealMystery() {}
+/* Declare a function named revealMystery that receives an envelope object (with the shape of the object returned by pickMystery) as the single argument, and returns a reveal message in the following format:
+
+<FIRST NAME> <LAST NAME> killed Mr. Boddy using the <WEAPON> in the <ROOM>!*/
+
```

Comment:
otra manera cool es usando bacticks 
return `${data.suspect.firstName} ${data.suspect.lastName} killed Mr. Boddy using the ${data.weapon.name} in the ${data.room.name}!`